docs/language/built-in-functions.md

```diff
@@ -31,3 +31,17 @@ title: Built-in Functions
 
   Follow [best practices](https://github.com/ConsenSys/smart-contract-best-practices/blob/051ec2e42a66f4641d5216063430f177f018826e/docs/recommendations.md#remember-that-on-chain-data-is-public)
   to prevent security issues when using this function.
+
+## rlpDecodeString
+
+`cadence•fun rlpDecodeString(input: [UInt8]): [UInt8]`
+
+  Accepts a RLP encoded byte array (called string in the context of RLP) and decodes it. 
+  Input should only contain a single encoded value for an string; if the encoded value type doesn't match or it has trailing unnecessary bytes it would error out. Any error while decoding fails the transaction. 
+
+## rlpDecodeList
```

Comment:
Same as above, please rename to `DecodeRLPList` for now to keep naming consistent.

Reply:
done